frontend/src/AlertsManagement/management_helpers.js

```diff
@@ -11,6 +12,8 @@ export async function getAlertsData(alertsLimit) {
   let unresolvedAlerts = [];
   let resolvedAlerts = [];
 
+  if (!alertsLimit) alertsLimit = MAX_ALERTS_LIMIT;
```

Comment:
you can probably use default parameters here? since were using new ecma script (ES6? im assuming)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters